package.json

```diff
@@ -29,7 +29,10 @@
     "preversion": "npm test && npm run build:js",
     "version": "npm run build:release && git add release/ && npm run update-doc-version && git add doc/version.py",
     "update-doc-version": "bash scripts/update-doc-version.sh",
-    "autobuild-docs": "cd doc && sphinx-autobuild . _build/html"
+    "autobuild-docs": "cd doc && sphinx-autobuild . _build/html",
```

Comment:
We should remove the build:docs and autobuild:docs scripts if we're not going to use them.